devops/terraform/host/OSConfigHost.tf

```diff
@@ -182,17 +182,3 @@ resource "azurerm_linux_virtual_machine" "osconfigvm" {
         environment = var.environment_tag
     }
 }
-
-resource "azurerm_dev_test_global_vm_shutdown_schedule" "vm_shutdown_schedule" {
```

Comment:
Does this change anything related to the cleanup of E2E resources?

Reply:
no, same azure resources, this is just an Azure Policy on the hosts we create